cli/cmd/testdata/install_output.golden

```diff
@@ -792,6 +792,8 @@ spec:
           value: ns:$(_pod_ns)
         - name: LINKERD2_PROXY_IDENTITY_DISABLED
           value: disabled
+        - name: LINKERD2_PROXY_TAP_SVC_NAME
+          value: linkerd-tap.$(_l5d_ns).serviceaccount.identity.$(_l5d_ns).$(_l5d_trustdomain)
```

Comment:
non-actionable observation: i would have expected `LINKERD2_PROXY_TAP_SVC_NAME` to already exist here prior to this PR, but glad to see it's here now.